### DISCUSSION TITLE
Maxium number of clients reached, with only one peer

### DISCUSSION THREAD

**Discussion by fscheps** [wg-easy/wg-easy#949] [Q&A]
Hi Community, first let me apologize for anything wrong I might say, I am learning linux from scratch with the help of our digital common assistant ChatGPT.

I installed before Wireguard standalone, but its a bit annoying to configure peers on wg.conf file so I decided to give it a try to wg-easy GUI.

The problem I am facing I think relies on the IP range definition I want to use. And the documentation on wg-easy doesn't seem to explicitly mention how this parameter should be used (although probably evident to anyone using Linux for years) :) 

This is how I am looking to set it up.

sudo docker run -d \
  --name wg-easy \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  -p 51820:51820/udp \
  -p 51821:51821/tcp \
  -v /etc/wireguard:/etc/wireguard \
  -e WG_HOST=192.168.1.100 \
  -e PASSWORD='MYWGEASYPASSWORD' \
  -e WG_DEFAULT_ADDRESS="192.168.160.1" \
  -e WG_DEFAULT_ADDRESS_RANGE="24" \
  --restart=unless-stopped \
  ghcr.io/wg-easy/wg-easy

But when I go to the GUI I get this error:
![image](https://github.com/wg-easy/wg-easy/assets/48051505/c90238c2-259f-4e60-9fa1-8966084475e7)

I want all my peers to obtain an address from range: 192.168.160.x and I want all my peers to be able to see each other, and also interact with 192.168.1.100 which is my Linux Server.

Logically I would want that whatever I define here is not overwritten by an update of WG-easy, and I would like to also ensure I can backup the configuration in case I need to restore.

Thanks a lot for your help!

Fernando

**Comment by eandersons** [comment]
`WG_DEFAULT_ADDRESS="192.168.160.1"` should be replaced with `WG_DEFAULT_ADDRESS="192.168.160.x"`.
